test/test_cli.rb

```diff
@@ -313,11 +313,20 @@ def test_load_path
     $LOAD_PATH.shift
   end
 
-  def test_environment
+  def test_environment_rack_env
     ENV.delete 'RACK_ENV'
 
     Puma::CLI.new ["--environment", @environment]
 
-    assert_equal ENV['RACK_ENV'], @environment
+    assert_equal @environment, ENV['RACK_ENV']
```

Comment:
This improves the output on failing  - the order of arguments is: `assert_equal exp, actual`